src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1682,8 +1692,10 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg,
                                      (char *)NULL)) {
     } else if (ceph_argparse_flag(args, i, "--pretty-format", (char *)NULL)) {
       cfg->pretty_format = true;
+    } else if (ceph_argparse_flag(args, i, "--legacy", (char *)NULL)) {
+      cfg->use_ioctl = true;
     } else if (ceph_argparse_flag(args, i, "--try-netlink", (char *)NULL)) {
-      cfg->try_netlink = true;
+      *err_msg << "rbd-nbd: --try_netlink is deprecated (already default)!";
```

Comment:
I think you still need to update `rbd_nbd` function to check if `err_msg` is empty after `parse_args` successful call, and if it is not empty, output to stderr. Otherwise this message will be just ignored.

Reply:
Make sense, done.